cmd/kubeops/main_test.go

```diff
@@ -123,6 +129,30 @@ func TestParseClusterConfig(t *testing.T) {
 ]`,
 			expectedErr: true,
 		},
+		{
+			name: "errors if more than one cluster with isKubeappsCluster=true is configured",
+			configJSON: `[
+		       {"name": "cluster-1", isKubeappsCluster: true},
+		       {"name": "cluster-2", isKubeappsCluster: true }
+		]`,
+			expectedErr: true,
+		},
+		{
+			name: "errors if no KubeappsClusterName can be resolved",
```

Comment:
Similar here: I don't think this should pass :)

Reply:
Which one? `: "errors if no KubeappsClusterName can be resolved"` ?
If we only specify:
```
 {"name": "cluster-1", "apiServiceURL": "https://example.com" }
 {"name": "cluster-2", "apiServiceURL": "https://example.com" }
```
We're not able to know which cluster is the correct one, are we?

Reply:
> Which one? `: "errors if no KubeappsClusterName can be resolved"` ?

Yes.

> If we only specify:
> 
> ```
>  {"name": "cluster-1", "apiServiceURL": "https://example.com" }
>  {"name": "cluster-2", "apiServiceURL": "https://example.com" }
> ```
> 
> We're not able to know which cluster is the correct one, are we?

Correct, none of the configured clusters for the user to select are the one on which Kubeapps is installed.